rockr/views.py

```diff
@@ -97,3 +109,24 @@ def get_user_role():
 def get_roles():
     resp = uq.get_roles()
     return format_response(resp["status"], resp["data"])
+
+@app.route('/matches', methods=["GET"])
+def get_matches():
+    user = User.query.filter_by(email=request.args.get("email")).first();
+    matches = db.session.query(User, UserMatch).join(User, User.id == UserMatch.user_id).filter(UserMatch.match_id == user.id).all()
```

Comment:
![image](https://github.com/LarrySchultheis/rockr-server/assets/44590233/5726aa01-5d99-40d4-a3af-540d88144382)

Reply:
The fact that you went through the time to make that meme says a lot. Will refactor lmao